### ISSUE TITLE
Ecology editors

### ISSUE THREAD

**Issue by rougier** [ReScience/ReScience#87]
@khinsen @benoit-girard @oliviaguest We may have to recruit new editors for ecology because we have regular submissions. I think @tpoisot proposed some names some time ago but I don't remember if I've contacted them. Else, we can promote reviewers that have already reviewed ecology submission. What do you think ?

**Comment by oliviaguest** [comment]
Why not compose a list of names (either way) and ask them if they want to help us?

**Comment by rougier** [comment]
Yes, we can do that. Maybe privately? I'm afraid a public list would put too much pressure on potential candidates.
